api/propagation/noop_propagator.go

```diff
@@ -0,0 +1,34 @@
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package propagation
+
+import (
+	"context"
+)
+
+type NoopPropagator struct {
+}
+
+var _ HTTPPropagator = NoopPropagator{}
+
+func (p NoopPropagator) Extract(ctx context.Context, supplier HTTPSupplier) context.Context {
+	return nil
```

Comment:
This should return the input context.  Otherwise, any other information that was in the context would be lost.

```suggestion
	return ctx
```